cmd/vpn_server/app/firewall.go

```diff
@@ -7,6 +7,7 @@ package app
 import (
 	"errors"
 	"fmt"
+	"github.com/gardener/vpn2/pkg/constants"
```

Comment:
Please separate at least standard library from the rest. Usually, we even separate other repositories from this repository, but that seems to be not the case in `vpn2`.

Reply:
yes. we don't have linters configured for vpn2 also. I suggest to add a .golangci-lint.yaml similar to [shoot-networking-filter](https://github.com/gardener/gardener-extension-shoot-networking-filter/blob/master/.golangci.yaml)

Reply:
I've added [a commit](https://github.com/gardener/vpn2/pull/137/commits/44da19df1cc5cffd70ba4bc1ef939a4d2b2932b9) to add a proper golangci.yaml and sorted the imports accordingly.